### ISSUE TITLE
Parallel data generation: controlling the number of parallel processes

### ISSUE THREAD

**Issue by GuillaumeBroggi** [bessagroup/f3dasm#271]
I might have missed it in the docs but it seems impossible to control the number of parallel jobs when generating data,

**Comment by mpvanderschelling** [comment]
Yes, in the current version 1.5.1 this is hard-coded to the number of cores you'll have available.
We can make this more flexible in the future

**Comment by GuillaumeBroggi** [comment]
This would be an important feature because one might want to run multiple tasks on a computer while parallelizing the data-driven workflow.
